sfcli/SmartFace.Cli/Core/Domain/ImgExport/Impl/ExportFileResolver.cs

```diff
@@ -43,7 +43,7 @@ private static string[] ParseDynamicProperties(string format)
         {
             var regex = new Regex(@"(?<=\{)[^}]*(?=\})");
             var matchCollection = regex.Matches(format);
-            var formatProperties = matchCollection.Select(mc => mc.Groups.First().Value).ToArray();
+            var formatProperties = matchCollection.Select(mc => mc.Groups.First<Group>().Value).ToArray();
```

Comment:
pseudochange?

Reply:
this was needed due to migrating to dotnet 3.1, as the `GroupCollection` started implementing the `IReadOnlyDictionary<string, Group>` interface, so linq isn't able to infer the type parameter anymore